osu.Game.Rulesets.Mania/osu.Game.Rulesets.Mania.csproj

```diff
@@ -35,11 +35,9 @@
   <ItemGroup>
     <Reference Include="nunit.framework, Version=3.8.1.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
       <HintPath>$(SolutionDir)\packages\NUnit.3.8.1\lib\net45\nunit.framework.dll</HintPath>
-      <Private>True</Private>

```

Comment:
What's with the inconsistent changes to these Private attributes? E.g. Private is removed from osu.Game reference in the catch ruleset, but is left in the taiko, osu!, and mania rulesets.

Do these need to even be removed at all? I would expect only the osu.Game.Tests one to need removal if anything.

Reply:
This PR was mostly a check to see if these changes would work on CI (since I can't test it without making a PR).  I'm not sure if this is the correct long term solution, anyway.